draft-ietf-quic-ack-frequency.md

```diff
@@ -338,25 +351,23 @@ Unreported Missing:
   have not yet been received.
 
 An endpoint that receives an ACK_FREQUENCY frame with a non-zero Reordering
-Threshold value SHOULD send an immediate ACK when the gap
-between the smallest Unreported Missing packet and the Largest Unacked is greater
-than or equal to the Reordering Threshold value. Sending this additional ACK will
-reset the `max_ack_delay` timer and `Ack-Eliciting Threshold` counter as any ACK
-would do.
+Threshold value SHOULD send an immediate ACK in response to an Out-of-Order
+Packet:
 
-In order to ensure timely loss detection, it is optimal to send a Reordering
-Threshold value of 1 less than the packet threshold used by the data sender for
-loss detection. If the threshold is smaller, an ACK_FRAME is sent before the
-packet can be declared lost based on the packet threshold. If the value is
-larger, it causes unnecessary delays. ({{Section 18.2 of QUIC-RECOVERY}})
-recommends a default packet threshold for loss detection of 3, equivalent to
-a Reordering Threshold of 2.
+* when the packet number is larger than the Largest Unacked
+  packet and the difference between the this packet number and the Largest
+  Unacked is larger than the Reordering Threshold.
 
-If the most recent ACK_FREQUENCY frame received from the peer has a `Reordering
-Threshold` value of 0, the endpoint SHOULD NOT send an immediate
-acknowledgement in response to packets received out of order, and instead
-rely on the peer's `Ack-Eliciting Threshold` and `max_ack_delay` thresholds
-for sending acknowledgements.
+* when the packet number is lower than the Largest Acked.
+  (TODO: is that true? Should there be some kind of threshold?)
```

Comment:
I think this is already true in RFC9000 and I don't think we were intending to change that.

Reply:
OK. I can remove this note.